services/QuillLMS/engines/evidence/Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
```

Comment:
why is there a Gemfile.lock diff but no Gemfile changes?

Reply:
Good question.  For some reason the version for evidence was out of sync with this [file](https://github.com/empirical-org/Empirical-Core/blob/513f60d4817e82fe7b0ef142551c4d06f5dc39ec/services/QuillLMS/engines/evidence/lib/evidence/version.rb#L6)